config/routes.rb

```diff
@@ -4,6 +4,8 @@
   namespace :api do
     post 'login', to: 'users#login'
     get 'feed', to: 'scores#user_feed'
-    resources :scores, only: %i[create destroy]
+    get 'scores', to: 'scores#user_feed'
+    get '/scores', to: 'scores#index'
+    get '/scores/:id', to: 'scores#show_scores'
```

Comment:
Routes defined here should reflect what's implemented on the controller.

It looks like we removed the routing for `create` and `destroy` on scores, but the methods are still defined in the controller.

Also, you defined the `index` route, but it is not implemented in the controller.